library/string.lisp

```diff
@@ -207,6 +227,13 @@ does not have that suffix."
                  (Err (concat "Cannot parse string as integer: " s))
                  (Ok z))))))
 
+  (define-instance ((Into :a String) (Into :b String) => (Into (Tuple :a :b) String))
```

Comment:
I think this is probably a net negative, and could trip one up in debugging.

Reply:
Removed.